iina/KeyBindingTranslator.swift

```diff
@@ -7,9 +7,106 @@
 //
 
 import Foundation
+import Mustache
 
 class KeyBindingTranslator {
 
+  static let l10nDic: [String: String] = {
+    let filePath = Bundle.main.path(forResource: "KeyBinding", ofType: "strings")!
```

Comment:
I tend to shy away from force unwrapping; this will be the first of these comments

Use a `guard` here